packages/nx-plugin/src/executors/autorun/schema.json

```
@@ -0,0 +1,100 @@
+{
+  "$schema": "http://json-schema.org/schema",
+  "$id": "AutorunExecutorOptions",
+  "title": "Add CodePushup configuration to a project",
+  "description": "Add CodePushup configuration to a project",
+  "type": "object",
+  "properties": {
+    "project": {
+      "type": "string",
+      "description": "The name of the project.",
+      "x-prompt": "Which project should configure Code Pushup?",
+      "x-dropdown": "projects",
+      "$default": {
+        "$source": "argv",
+        "index": 0
+      }
+    },
+    "dryRun": {
+      "type": "boolean",
+      "description": "Print the commands that would be run, but don't actually run them"
+    },
+    "verbose": {
+      "type": "boolean",
+      "description": "Print additional logs"
+    },
+    "progress": {
+      "type": "boolean",
+      "description": "Print additional logs"
+    },
+    "onlyPlugins": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "description": "Only run the specified plugins"
+    },
+    "config": {
+      "type": "string",
+      "description": "Path to the configuration file"
+    },
+    "projectPrefix": {
+      "type": "string",
+      "description": "Prefix for project name (default is ``)"
+    },
+    "persist": {
+      "type": "object",
+      "properties": {
+        "filename": {
+          "type": "string",
+          "description": "Filename to save the configuration to (default is `.codepushup.json`)"
+        },
+        "outDir": {
```

Why give it a different name than in our config?

```suggestion
        "outputDir": {
```

---

OMG this was a mistake. In future a PR I will autogenerate the schema.json from types.